__tests__/app.test.js

```diff
@@ -108,4 +108,67 @@ describe('GET /api', () => {
             });
         });
     })
-})
+});
+describe('GET /api/articles/:article_id/comments', () => {
+  describe('Basic request checks', () => {
```

Comment:
what does "Basic request checks" mean? they should be describing behaviour, not inferring I know something about this project. Remember a test suite is a form of documentation.